src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -34,7 +34,16 @@ public void HandleApplicationEvents(Application application, bool bind = true)
     {
         if (bind)
         {
-            // Attach element events to all descendents as they are added to the application.
+            // Attach element events to all existing descendants.
+            foreach (var descendant in application.GetVisualTreeDescendants().Skip(1))
```

Comment:
Just wondering why the `Skip(1)`? Maybe worth adding a comment to explain that...